Scripts/Editor/AssemblyInjections/UnityEditor/ConsoleHelper.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Reflection;
+using UnityEditor;
+using UnityEngine;
+
+namespace Anvil.Unity.Editor.Debug
```

Comment:
Just noticed this and we should push this off to another PR but should we move all of this namespace to `Debugging` to avoid conflict with the `Debug` class?

That's what we did in the runtime code.

Reply:
Yep, done!

Reply:
Sorry I meant all of the classes under this namespace `ScriptDefinesToggle`, `ScriptDefineDefinition`, etc...

Let's do that as part of a new PR though since this one is good to go.

Reply:
Cool, tracked in issue here: https://github.com/decline-cookies/anvil-unity-core/issues/118